CONTRIBUTING.md

```diff
@@ -107,7 +107,7 @@ When an underlying runtime like [rive-ios](https://github.com/rive-app/rive-ios)
 Before running `yarn` at the root level of the project, set the appropriate version of the iOS runtime in `rive-react-native.podspec` at the root of the folder:
 
 ```
-s.dependency "RiveRuntime", "5.1.6"
+s.dependency "RiveRuntime", "6.2.0"
```

Comment:
We now have v6.2.1 which fixes some potentials for crashes on iOS, might want to bump the bump!

Reply:
Thanks. Added the newer version. I also make this section of the docs generic (`x.x.x`) so we don't have to keep updating it every time.